universe/common/api/middleware/verify_user.go

```diff
@@ -7,27 +7,23 @@ import (
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 
-	"github.com/momentum-xyz/ubercontroller/types/entry"
+	"github.com/momentum-xyz/ubercontroller/types/generic"
 	"github.com/momentum-xyz/ubercontroller/universe"
 	"github.com/momentum-xyz/ubercontroller/universe/common/api"
-	"github.com/momentum-xyz/ubercontroller/utils"
 )
 
+var secretStore = generic.NewSyncMap[string, *[]byte](0)
```

Comment:
for me it's a bit complicated
I suggest just load the secret into private variable in VerifyUser() before returning gin.HandlerFunc